updatecli/update-cli-ref-docs.yaml

```diff
@@ -0,0 +1,6 @@
+policies:
+  - name: Update kwctl CLI reference documentation
+    policy: ghcr.io/jvanz/updatecli/policies/file:0.2.0
```

Comment:
This must be updated when the policy got accepted in the updatecli org

https://github.com/updatecli/policies/pull/41